hw03_rileyj3.py

```diff
@@ -0,0 +1,120 @@
+######################################################################
+# Author: Bryce Riley
+# Username: rileyj3
+#
+# Assignment: HW03 : Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To create something complex and learn about GitHub.
+# Link to Document: https://docs.google.com/document/d/1EzPApStfIckqskXxnh8CNhMuNgis_GTE2FZZHHZvji4/edit?usp=sharing
+#####################################################################
+
+
+import turtle
+
+bryce = turtle.Turtle()
```

Comment:
You've still got code outside of main. Move bryce into main, then pass him into functions:

```
def build_house(b):
    b.forward(10) 
    #...


def main():
    bryce = turtle.Turtle()
    build_house(bryce)
```